WordImpress/Give-iATS
# Fix: fatal error php 8 1

Resolves https://github.com/impress-org/give-iats/issues/24

## Description


I found that [`inacho/php-credit-card-validator`](https://packagist.org/packages/inacho/php-credit-card-validator) library is not compatible with PHP 8.1, so I am using `cweagans/composer-patches` to patch library to resolve the compatibility issue.

## Affects


This should affect donation processing.

## Testing Instructions


- Donor should be able to process donation without any error on PHP 7.0 to 8.1

**Note: You should have `gpatch`  installed. You can install this application with `brew install gpatch`  command.**

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205485182014696
